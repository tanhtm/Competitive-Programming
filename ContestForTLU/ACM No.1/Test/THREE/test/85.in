558352200000000000
