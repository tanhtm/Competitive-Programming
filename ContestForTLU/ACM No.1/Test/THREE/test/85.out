4 5 8 10 12 13 14 16 17 21 26 28 35 
1 6 9 18 22 25 31 32 33 36 37 
